core/encoding/json/doc.odin

```diff
@@ -0,0 +1,57 @@
+/*
+The `encoding/json` package can be used to parse and validate JSON files. A simple example is shown below.
+
+`test.json`
+<pre><code class="hljs" data-lang="json">
```

Comment:
No HTML tags. 

This is better solved with the documentation generation tool rather than just hacking it in.